### DISCUSSION TITLE
html_show_copyright causes empty section in footer

### DISCUSSION THREAD

**Discussion by fuglede** [pradyunsg/furo#220] [Potential Issue] [answered]
If I let `html_show_copyright = False`, my footer looks like this:

![image](https://user-images.githubusercontent.com/6169306/130216873-ee0424b7-b24e-4d42-a18e-fa00a5be01c6.png)

The first pipe seems like something that shouldn't be there.

**Comment by jdknight** [comment, marked as answer]
To be addressed in https://github.com/pradyunsg/furo/pull/225.
